src/pages/users/[id].tsx

```diff
@@ -15,7 +16,37 @@ interface Props {
     models: Record<ModelId, Model>;
 }
 
-export default function Page({ userId, user, models }: Props) {
+const fixDescription = (description: string, scale: number) => {
```

Comment:
I was more commenting on the fact that you copied this from `index.tsx`... I think that we should fix this code duplication.

Reply:
Oh whoops. That's what i thought you meant but i just completely forgot to replace both when i actually went to go do it lol